### PR TITLE
Mw feature submitreview

### PR DESCRIPTION
**Pull request by MattWaraich**
#Review Page
- Style Review Page
- Add star rating
- Add input form for review submission
- Call data from database
- Edit review section (Event page)

### CODE REVIEW

File: client/js/components/ReviewScore/ReviewScore.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Image, View} from 'react-native';
+import styles from './styles';
+
+export default ReviewScore = ({score}) => {
```
Comment (siddharthparmar7) [inline client/js/components/ReviewScore/ReviewScore.js]: Make sure to rebase properly. This file has been changed in the Develop branch to remove bugs.

File: client/js/screens/SubmitReview/SubmitReview.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import {Field, FormSpy, Form} from 'react-final-form';
+import {Mutation} from 'react-apollo';
+import {MUTATION_ADD_REVIEW} from '../../apollo/mutations';
+
+const SubmitReview = ({navigation, eventInfo}) => {
+  return (
+    <Mutation mutation={MUTATION_ADD_REVIEW}>
+      {(addReview, {loading, error, data}) => (
+        <>
+          <SafeAreaView>
```
Comment (siddharthparmar7) [inline client/js/screens/SubmitReview/SubmitReview.js]: Remove this SafeAreaView to avoid style bugs.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: client/js/components/ReviewScore/ReviewScore.js (hunk 1 above)
Comment (MattWaraich) [reply client/js/components/ReviewScore/ReviewScore.js]: Understood.

File: client/js/screens/SubmitReview/SubmitReview.js (hunk 2 above)
Comment (MattWaraich) [reply client/js/screens/SubmitReview/SubmitReview.js]: Complete.

File: client/js/components/ReviewScore/ReviewScore.js (hunk 1 above)
Comment (siddharthparmar7) [reply client/js/components/ReviewScore/ReviewScore.js]: change `export default ReviewScore = ({score}) => {` to `const ReviewScore = ({score}) => {`

and at the end of your file

`export default ReviewScore`

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**